### DISCUSSION TITLE
Can you add Google adsense to the tailwind-nextjs-starter-blog?

### DISCUSSION THREAD

**Discussion by Abinaskarki** [timlrx/tailwind-nextjs-starter-blog#635] [General]
I tried adding adsense script in the _document.js  Head tag, however, google ads are not showing. 

I also removed the Content-Security-Policy from next.config.js. But, the issue prevails. 

Has anyone enabled Google Adsense for the tailwind-nextjs-starter-blog?

**Comment by Abinaskarki** [comment]
Yes, you can add Google adsense to this blog by adding adsense script in the Head of _document.js file on pages folder like below. 

          <script
            async
            src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js?client=ca-pub-xxxxxxxxxxxxx"
            crossOrigin="anonymous"
          ></script>

Also, it is better to add ads.txt file with adsense credentials you get from Google Adsense in the public folder.

**Reply by kkian481718** [reply]
bro i just cannot find the "_document.js" file : (
Is it being removed in v2?
